src/overlays/actors/ovl_En_Po_Composer/z_en_po_composer.h

```diff
@@ -2,16 +2,59 @@
 #define Z_EN_PO_COMPOSER_H
 
 #include "global.h"
+#include "objects/object_po_composer/object_po_composer.h"
+
+#define PO_COMPOSER_IS_FLAT(actor) ((actor)->params & 0x8000)
+#define PO_COMPOSER_4000(actor)    ((actor)->params & 0x4000)
```

Comment:
```suggestion
#define PO_COMPOSER_IS_FLAT(thisx) ((thisx)->params & 0x8000)
#define PO_COMPOSER_4000(thisx)    ((thisx)->params & 0x4000)
```
Small nitpick, but so its consistent with the other actors.